bears/natural_language/LanguageToolBear.py

```diff
@@ -33,16 +33,16 @@ def check_prerequisites(cls):
             except ImportError:  # pragma: no cover
                 return 'Please install the `language-check` pip package.'
 
-    @deprecate_settings(language='locale')
+    @deprecate_settings(natural_language='locale')
```

Comment:
argh that means that if the user used `language` before this won't work and he won't get a deprecation warning, that's not ok. I'm not sure if the decorator is capable of two deprecations of the same variable but we gotta figure it out :/